research/object_detection/exporter_main_v2.py

```diff
@@ -106,6 +106,27 @@
 flags.DEFINE_string('config_override', '',
                     'pipeline_pb2.TrainEvalPipelineConfig '
                     'text proto to override pipeline_config_path.')
+flags.DEFINE_boolean('use_side_inputs', False,
+                     'If True, uses side inputs as well as image inputs.')
+flags.DEFINE_string('side_input_shapes', "",
```

Comment:
If it tuns out that these are required, I would suggest writing an example spec in the helper string.